src/assets/scss/components/_imageinput.scss

```diff
@@ -0,0 +1,32 @@
+.image-input {
+    input[type=radio]:checked + .image-input__image {
+        border-radius: 10%;
+        box-shadow: inset 0 0 0 5px $accent; 
+        padding: 10px;
+        transition: all .5s ease-in-out;
+    }
+     input[type=radio]:checked ~ .image-input--text {
+        color: $accent;
+        transition: all .5s ease-in-out;
+    }
+    input[type=radio]:hover ~ .image-input--text{
+        color: $accent;
+        transition: all .5s ease-in-out;
+        cursor: pointer;
+    }
+    &__button {
+       opacity: 0;
+   }  
+    &__image {
+        height: 150px;
+        width: 150px;
+        padding: 10px;
+        &:hover {
+          border-radius: 10%;
+          box-shadow: inset 0 0 0 5px $accent; 
+          padding: 15px;  
+          transition: all .5s ease-in-out;
+          cursor: pointer;
```

Comment:
`cursor` will only ever be shown on `:hover` so you can add it to `&__image` rather than `&__image:hover`